### DISCUSSION TITLE
Refresh content when files in content/*.json,md... folder are modified

### DISCUSSION THREAD

**Discussion by johan-Rm** [nuxt/nuxt#16744] [Questions]
I'm using Nuxt 3 and would like to know if hot reloading data when the source is updated, in my case it's the changes on the filesystem of the '/content/*.json' folder using the @nuxt-content2 module.

To date in env=dev mode, the hot reloading of data on the page works very well, 

when will the same behavior occur in env=prod mode? Anyone have any news on this?

**Comment by TheAlexLichter** [comment]
There is no HMR in production mode ☺️

**Reply by johan-Rm** [reply]
yes to date there is no HMR but it seems that it is planned ;)

https://github.com/nuxt/content/issues/783#issuecomment-789027175
